src/main/java/de/tum/in/www1/exerciseapp/web/rest/ParticipationResource.java

```diff
@@ -74,7 +68,7 @@ public ParticipationResource(ParticipationService participationService, Particip
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
     @PostMapping("/participations")
-    @PreAuthorize("hasAnyRole('TA', 'ADMIN')")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
     @Timed
     public ResponseEntity<Participation> createParticipation(@RequestBody Participation participation) throws URISyntaxException {
```

Comment:
This should only be possible for TAs and instrcutors of this particular course (and admins)

Reply:
Where is this used? I suspect it was generated with JHipster and never used. Participations are created with POST /api/courses/:id/exercises/:id/participations.

Reply:
Done.